src/Apps/Fairs/Routes/FairsIndex.tsx

```diff
@@ -293,13 +293,9 @@ export const FairsIndex: React.FC<React.PropsWithChildren<FairsIndexProps>> = ({
             {upcomingFairs.map(fair => {
               return (
                 <Text key={fair.internalID} my={2} variant="sm">
-                  {fair.organizer?.profile?.href ? (
-                    <RouterLink inline to={fair.organizer.profile.href}>
-                      {fair.name}
-                    </RouterLink>
-                  ) : (
-                    fair.name
-                  )}
+                  <RouterLink inline to={fair.profile?.href ?? "#"}>
```

Comment:
Don't quite understand the "#" bit -- if there's no url, why still use RouterLink? (Also think the # could confuse the system, but not totally sure). 

All good tho if this is deliberate and expected, mostly just curious 👍

Reply:
Yah check out the diff, how it was -- that's the correct way, where if there's no href we just render text! The safest there.

Reply:
totally